crates/release_plz_core/src/command/release.rs

```diff
@@ -575,30 +580,32 @@ async fn should_release(
 fn registry_indexes(
     package: &Package,
     registry: Option<String>,
-) -> anyhow::Result<Vec<CargoIndex>> {
+) -> anyhow::Result<Vec<(Option<String>, CargoIndex)>> {
```

Comment:
could you turn the tuple into a struct? This type is already complex, thanks!

Reply:
Sure, I was not sur if you prefer:
- the tuple
- a new struct which name? (as registry is a String or something else define in other crates)

```rust
struct CargoRegistry {
  name: Option<String>,
  index: CargoIndex,
}
```

Reply:
I like your `CargoRegistry` proposal 👍

I prefer the struct as it's more explicit